README.md

```diff
@@ -10,22 +10,26 @@ This is an exercise to check your knowledge on using secrets in GitHub Actions w
 
 :information_source: The use of GitHub Actions also means that it may take the grading workflow a few seconds and sometimes minutes to run.
 
+:closed_lock_with_key: This exercise asks you to create a personal access token (PAT) with `repo` scope. This step is a prerequisite of the exercise, and you should already be comfortable with PATs on GitHub before proceeding. The PAT is encrypted and used only by the local action [`check-secrets-action`](.github/actions/check-secrets-action) and does not leave your repository. GitHub has safeguards in place if the PAT is exposed by mistake, but you should know that if someone gains access to your PAT they would have access to your GitHub repositories. To be safe, we recommend destroying or regenerating the token once you've completed this exercise. See _[Personal Access Token - GitHub Docs]_ and _[`repo` scope]_ for more information.
```

Comment:
Addresses some of the feedback in https://github.com/github/programs-forge/issues/159#issuecomment-849007878. 

@mattdavis0351 would you put some 👀 on this from a correctness and security lens?

@lucie-docs feel free to move this section where appropriate, but wanted to get some of this language in to address the feedback.

Reply:
Looks good to me where it is right now 🙂